mypy/semanal.py

```diff
@@ -2828,7 +2828,16 @@ def visit_file(self, file: MypyFile, fnam: str, mod_id: str, options: Options) -
 
         # Add implicit definitions of module '__name__' etc.
         for name, t in implicit_module_attrs.items():
-            v = Var(name, UnboundType(t))
+            # Issue #2471: unicode docstrings should be accepted in Python 2
```

Comment:
It's not necessary to mention the issue number (but the rest of the comment is useful).